xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources_ru.properties

```diff
@@ -221,6 +221,12 @@ XWiki.XWikiPreferences_showhistory=\u041F\u043E\u043A\u0430\u0437\u044B\u0432\u0
 XWiki.XWikiPreferences_showinformation=\u041F\u043E\u043A\u0430\u0437\u044B\u0432\u0430\u0442\u044C \u0438\u043D\u0444\u043E\u0440\u043C\u0430\u0446\u0438\u044E \u043E \u0434\u043E\u043A\u0443\u043C\u0435\u043D\u0442\u0435
 XWiki.XWikiPreferences_skin=\u0421\u043A\u0438\u043D
 XWiki.XWikiPreferences_smtp_port=\u041F\u043E\u0440\u0442
+XWiki.XWikiPreferences_ldap_update_photo=\u041e\u0431\u043d\u043e\u0432\u0438\u0442\u044c \u0444\u043e\u0442\u043e\u0433\u0440\u0430\u0444\u0438\u044e \u043f\u043e\u043b\u044c\u0437\u043e\u0432\u0430\u0442\u0435\u043b\u044f \u0447\u0435\u0440\u0435\u0437 LDAP
```

Comment:
The reference for translations is http://l10n.xwiki.org, anything you put here will be overwritten at nest release.